docs/src/capi/tutorial/getting-started.md

```diff
@@ -94,7 +94,49 @@ export AWS_B64ENCODED_CREDENTIALS=$(clusterawsadm bootstrap credentials encode-a
 ```
 
 You are now all set to deploy the AWS CAPI infrastructure provider.
+````
+
+````{group-tab} MAAS
+Start by setting up environment variables defining the MAAS credentials, if
+these are not already defined:
+
+```
+export MAAS_API_KEY="<maas-api-key>"
+export MAAS_ENDPOINT="http://<maas-endpoint>/MAAS"
+export MAAS_DNS_DOMAIN="<maas-dns-domain>"
+```
+The MAAS infrastructure provider uses these credentials to deploy machines, create DNS records and perform various other operations for workload clusters.
+
+```{warning}
+The management cluster needs to be able to resolve dns records from the MAAS domain.
+This usually means the management cluster also needs to be deployed on a MAAS machine.
+```
+
+Continue with setting up environment variables defining the machine image and minimum compute resources of the control plane and worker nodes:
+
+```
+export CONTROL_PLANE_MACHINE_MINCPU="2"
+export CONTROL_PLANE_MACHINE_MINMEMORY="4096"
+export CONTROL_PLANE_MACHINE_IMAGE="ubuntu"
+
+export WORKER_MACHINE_MINCPU="2"
+export WORKER_MACHINE_MINMEMORY="4096"
```

Comment:
I think 4G of memory is a bit high. I'd set it to 2048 by default and the user can choose if they want more.